bookkeeper-server/src/test/java/org/apache/bookkeeper/client/MockBookKeeper.java

```diff
@@ -210,6 +213,66 @@ public void close() throws InterruptedException, BKException {
         shutdown();
     }
 
+    @Override
+    public OpenBuilder newOpenLedgerOp() {
+        return new OpenBuilder() {
```

Comment:
why not extending `OpenBuilderImpl` because most of the methods here are builder methods. what you really need is overriding the `execute` method.